components/automate-ui/src/app/modules/infra-proxy/create-databag-item-modal/create-databag-item-modal.component.ts

```diff
@@ -123,14 +124,11 @@ export class CreateDatabagItemModalComponent implements OnInit, OnDestroy {
   }
 
   onChangeJSON(event: { target: { value: string } }) {
-    // get value from text area
     const newValue = event.target.value;
     try {
-      // parse it to json
       JSON.parse(newValue);
       this.itemAttrParseError = false;
     } catch (ex) {
-      // set parse error if it fails
       this.itemAttrParseError = true;
```

Comment:
Author note: Comments should only be present if they add value; these are self-evident, no value added. And having the comments makes the code 50% longer and harder to read and harder to maintain.